src/utils/vaults.ts

```diff
@@ -28,6 +28,7 @@ const VAULT_VIEW_METHODS = [
     'debtRatio',
     'totalDebt',
     'lastReport',
+    'rewards',
```

Comment:
we can add warning similar to governance and management address that needs to be treasury.ychad.eth
0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde else have a warning